### PR TITLE
`vttablet` program docs: add `--tx-throttler-tablet-types` flag

### PR DESCRIPTION
**Pull request by timvaillancourt**
This PR adds the `vttablet` flag `--tx-throttler-tablet-types` flag to the program documentation for ~~v16~~ v17

This flag is added in this PR https://github.com/vitessio/vitess/pull/12174

### CODE REVIEW

**Review by rsajwani — APPROVED** [review]

**Comment by rsajwani** [comment]
Please make sure this gets push only after the PR is merge.

**Comment by harshit-gangal** [comment]
This change should be done for the 17.0 doc and not for 16.0

Also, check https://vitess.io/docs/17.0/reference/features/tablet-throttler/ if this needs an update.

**Comment by harshit-gangal** [comment]
Sorry for not getting to this earlier to make it to 16.0

**Comment by timvaillancourt** [comment]
> Sorry for not getting to this earlier to make it to 16.0

@harshit-gangal no problem! Looks like this is ready now that https://github.com/vitessio/vitess/pull/12174 is merged 👍 🙇

File: content/en/docs/16.0/reference/programs/vttablet.md [hunk 1]
```diff
@@ -397,6 +397,7 @@ The following global options apply to `vttablet`:
...
 | --tx-throttler-healthcheck-cells                                 | strings       | Synonym to -tx_throttler_healthcheck_cells                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
+| --tx-throttler-tablet-types                                      | strings       | A comma-separated list of tablet types. Only tablets of this type are monitored for replication lag by the transaction throttler. Supported types are replica and/or rdonly. (default replica)                                                                                                                                                                                                                                                                                                             |
```
Comment (harshit-gangal) [inline content/en/docs/16.0/reference/programs/vttablet.md]: this would need to be changed in `content/en/docs/17.0/reference/programs/vttablet.md` now.

Comment (timvaillancourt) [reply content/en/docs/16.0/reference/programs/vttablet.md]: @harshit-gangal makes sense, updated!

**Review by harshit-gangal — APPROVED** [review]

**Review by shlomi-noach — APPROVED** [review]

**Pull request merged**